cmd/cpe/generate.go

```diff
@@ -20,11 +23,44 @@ func panicif(err error) {
 }
 
 func main() {
-	fmt.Println("Starting CPE sqlite generation")
+	var (
+		runCentOS bool
+		verbose   bool
+	)
+	flag.BoolVar(&runCentOS, "centos", true, "Sets whether to run the CentOS sqlite generation")
+	flag.BoolVar(&verbose, "verbose", false, "Sets verbose mode")
+	flag.Parse()
 
-	cwd, err := os.Getwd()
+	dbPath := cpe()
+
+	fmt.Printf("Sqlite file %s size: %.2f MB\n", dbPath, getSizeMB(dbPath))
+
+	// The CentOS repository data is added to the CPE database.
+	if runCentOS {
+		centos(dbPath, verbose)
+	}
+
+	fmt.Printf("Sqlite file %s size: %.2f MB\n", dbPath, getSizeMB(dbPath))
```

Comment:
If the `centos` flag was not passed, then we would end up printing duplicate lines.

```suggestion
	// The CentOS repository data is added to the CPE database.
	if runCentOS {
		centos(dbPath, verbose)
		fmt.Printf("Sqlite file %s size: %.2f MB\n", dbPath, getSizeMB(dbPath))
	}
```

Reply:
Good catch! Fixed!